homeassistant/helpers/entity.py

```diff
@@ -1496,6 +1499,31 @@ def _suggest_report_issue(self) -> str:
             self.hass, integration_domain=platform_name, module=type(self).__module__
         )
 
+    @callback
+    def _report_deprecated_supported_features_constants(
+        self, replacement: IntFlag
+    ) -> None:
+        """Report deprecated supported features constants."""
+        if self._deprecated_supported_features_reported is False:
+            return
+        self._deprecated_supported_features_reported = True
+        report_issue = self._suggest_report_issue()
+        report_issue += (
+            " and reference "
+            "https://developers.home-assistant.io/blog/2022/04/02/support-constants-deprecation/"
```

Comment:
This blog post ain't related

Reply:
Should I make a new one?

Reply:
We have never officially deprecated this, so yeah, if we plan on deprecating this, we need to announce it properly and set a deprecation period.